modules/todo.js

```diff
@@ -0,0 +1,31 @@
+const todos = [];
+
+function addTodos() {
+  const inputTitle = document.getElementById("title").value;
+  const inputDescription = document.getElementById("description").value;
+  const inputDate = document.getElementById("date").value;
+  console.log(inputTitle);
+
+  const todo = {
+    titel: inputTitle,
+    description: inputDescription,
+    date: inputDate,
+  };
+
+  todos.push(todo);
+  console.log(todos);
+  hide();
+}
+
+function show() {
+  document.getElementById("input-block").style.display = "block";
+}
+
+function hide() {
+  document.getElementById("input-block").style.display = "none";
+}
+
+const addButton = document.getElementById("addTodo-button");
+addButton.addEventListener("click", show);
```

Comment:
Ska vi ändra till en toggle så man kan stänga todo formuläret utan att lägga till en todo?

```suggestion
  toggleTodoForm();
}

let todoFormVisible = false;

function toggleTodoForm() {
  todoFormVisible = !todoFormVisible;
  let displayStyle = todoFormVisible ? "block" : "none";

  document.getElementById("input-block").style.display = displayStyle;
}

const addButton = document.getElementById("addTodo-button");
addButton.addEventListener("click", toggleTodoForm);
```